examples/ecs/load_dynamic_plugin.rs

```diff
@@ -0,0 +1,33 @@
+//! Loads a dynamic plugin from the shared library `dynamic_plugin`
+
+use std::path::PathBuf;
+
+use bevy::prelude::*;
+use bevy_dynamic_plugin::dynamically_load_plugin;
+use libloading::Library;
+use parking_lot::Mutex;
+
+/// If the library wasn't stored here, libloading would unload it as soon as we were done with it below.
+/// This would cause the program to crash/segfault later when bevy tries to invoke the plugin's systems.
+/// Therefore, libraries should be stored such that they outlive the [`App`].
+/// A simple method is to use a global mutex.
```

Comment:
would a `OnceCell` be better than a `Mutex` here?

Reply:
Yeah, in this instance, though if you have multiple libraries in a more complex type you probably need Mutex.